src/main/java/uk/gov/hmcts/reform/civil/documentmanagement/model/CaseDocument.java

```diff
@@ -18,6 +19,7 @@ public class CaseDocument {
     private final long documentSize;
     private final LocalDateTime createdDatetime;
     private final String createdBy;
+    private final CaseRole ownedBy;
```

Comment:
Does this need to be Optional<CaseRole>?

Reply:
Yes, it does.